secrets.test.js

```diff
@@ -26,17 +25,16 @@ test('export dynamic secrets', async () => {
         'token': 'akeyless-token',
         'name': '/path/to/dynamic/producer',
     });
-    expect(core.setSecret).toHaveBeenCalledWith('sup', dynamicSecret);
```

Comment:
You are calling setSecret - that's the part that makes it a secret, which is also the important part, so you may as well leave this in and test that both `setSecret` and `setOuptut` are called.

Reply:
I originally had all three setSecret, setOutput and exportVariable in the update. The same has occurred again.

did I miss something you intended in the test's setup?

![image](https://user-images.githubusercontent.com/3520532/187914160-2ff3f2d6-608a-4264-9575-485ffee65269.png)